### PR TITLE
create_object_from_face and create_object_from_edge now supports list of faces as input and also allows to create nonmodel objects.

### PR DESCRIPTION
**Pull request by maxcapodi78**
#874

### CODE REVIEW

File: pyaedt/modeler/Object3d.py [hunk 1]
```diff
@@ -1047,20 +1049,22 @@ def normal(self):
             return inv_norm
 
     @pyaedt_function_handler()
-    def create_object(self):
+    def create_object(self, non_model=False):
         """Return A new object from the selected face.
```
Comment (Samuelopez-ansys) [inline pyaedt/modeler/Object3d.py:1053]:
```suggestion
        """Return a new object from the selected face.
```

File: pyaedt/modeler/Object3d.py [hunk 2]
```diff
@@ -1047,20 +1049,22 @@ def normal(self):
...
 
         Returns
         -------
         :class:`pyaedt.modeler.Object3d.Object3d`
             3D object.
+        non_model : bool, optional
+            Either if create the new object as model or nonmodel. Default is `False`.
```
Comment (Samuelopez-ansys) [inline pyaedt/modeler/Object3d.py:1060]:
```suggestion
            Either if create the new object as model or nonmodel. The default is ``False``.
```

**Review by Samuelopez-ansys — CHANGES_REQUESTED** [review]

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/modeler/Object3d.py [hunk 3]
```diff
@@ -534,20 +534,22 @@ def __str__(self):
...
-    def create_object(self):
+    def create_object(self, non_model=False, create_group_for_new_objects=False):
         """Return A new object from the selected edge.
 
         Returns
         -------
         :class:`pyaedt.modeler.Object3d.Object3d`
             3D object.
+        non_model : bool, optional
+            Either if create the new object as model or nonmodel. Default is `False`.
```
Comment (MaxJPRey) [inline pyaedt/modeler/Object3d.py:545]:
```suggestion
            Either if create the new object as model or non-model. Default is ``False``.
```

File: pyaedt/modeler/Object3d.py [hunk 4]
```diff
@@ -534,20 +534,22 @@ def __str__(self):
         return "EdgeId " + str(self.id)
 
     @pyaedt_function_handler()
-    def create_object(self):
+    def create_object(self, non_model=False, create_group_for_new_objects=False):
         """Return A new object from the selected edge.
```
Comment (MaxJPRey) [inline pyaedt/modeler/Object3d.py:538]:
```suggestion
        """Return a new object from the selected edge.
```

File: pyaedt/modeler/Object3d.py [hunk 5]
```diff
@@ -534,20 +534,22 @@ def __str__(self):
...
-        """Return A new object from the selected edge.
+    def create_object(self, non_model=False, create_group_for_new_objects=False):
+        """Return a new object from the selected edge.
 
         Returns
         -------
         :class:`pyaedt.modeler.Object3d.Object3d`
             3D object.
+        non_model : bool, optional
+            Either if create the new object as model or non-model. Default is `False`.
```
Comment (maxcapodi78) [inline pyaedt/modeler/Object3d.py:545]:
```suggestion
            Either if create the new object as model or non-model. The default is `False`.
```

File: pyaedt/modeler/Primitives.py [hunk 6]
```diff
@@ -1304,77 +1303,105 @@ def create_region(self, pad_percent=300):
...
+        """Create an object from one or multiple edges.
 
         Parameters
         ----------
-        edge : int or :class:`pyaedt.modeler.Object3d.EdgePrimitive`
-            Edge ID or :class:`pyaedt.modeler.Object3d.EdgePrimitive` object.
+        edge : list, int or :class:`pyaedt.modeler.Object3d.FacePrimitive`
+            Face ID or :class:`pyaedt.modeler.Object3d.FacePrimitive` object or Face List.
+        non_model : bool, optional
+            Either if create the new object as model or non-model. Default is `False`.
```
Comment (maxcapodi78) [inline pyaedt/modeler/Primitives.py:1315]:
```suggestion
            Either if create the new object as model or non-model. The default is `False`.
```

File: pyaedt/modeler/Object3d.py (hunk 3 above)
Comment (maxcapodi78) [reply pyaedt/modeler/Object3d.py]: done thanks.

File: pyaedt/modeler/Object3d.py (hunk 2 above)
Comment (maxcapodi78) [reply pyaedt/modeler/Object3d.py]: Done

**Pull request merged**